packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.tsx

```diff
@@ -0,0 +1,472 @@
+import React, {
```

Comment:
This is based on the `routes/Product/SubscriptionCreate` component, but getting a significant overhaul to accommodate the new retry and confirmation flows.